src/Acuminator/Acuminator.Utils/RoslynExtensions/PXContext.cs

```diff
@@ -150,6 +143,33 @@ public FieldAttributesTypes(Compilation aCompilation)
             #endregion
         }
 		#endregion
+		#region Attributes Types
+		public class AttributesTypes
+		{
+			private readonly Compilation compilation;
```

Comment:
We agreed that underscores should be used for private members. See [this article](https://wiki.acumatica.com/display/ENG/Acuminator+team+code+style).